packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -251,32 +253,42 @@ function ItemsPerPageControl() {
 	);
 }
 
-function FieldItem( {
-	field,
+function BaseFieldItem( {
+	fieldId,
+	label,
+	subLabel,
 	isVisible,
 	isFirst,
 	isLast,
 	canMove = true,
+	canHide = true,
+	isInteracting = false,
 	onToggleVisibility,
 	onMoveUp,
 	onMoveDown,
+	additionalActions,
 }: {
-	field: NormalizedField< any >;
+	fieldId: string;
+	label: string;
+	subLabel?: string;
```

Comment:
Maybe `description` or something like that is more fitting? `subLabel` is just confusing and has a very specific different meaning in English.

Reply:
That's good point it was updated.